tests/envs/mujoco/test_mujoco_custom_env.py

```diff
@@ -116,3 +119,11 @@ def test_xml_file():
     assert env.unwrapped.data.qpos.size == 9
 
     # note can not test user home path (with '~') because github CI does not have a home folder
+
+
+def test_reset_info():
+    """Verify that the environment returns info at `reset()`"""
+    env = PointEnv()
+
+    _, info = env.reset()
+    assert info["works"]
```

Comment:
`is True` , there are many things that `assert a` is true

Reply:
Done